cmd/cmd.go

```diff
@@ -76,14 +76,14 @@ func init() {
 	flag.Set("logtostderr", "true")
 
 	driverCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", kubeconfig, "path to kubeconfig")
-	driverCmd.PersistentFlags().StringVarP(&identity, "identity", "i", identity, "identity of this direct-csi")
-	driverCmd.PersistentFlags().StringVarP(&endpoint, "endpoint", "e", endpoint, "endpoint at which direct-csi is listening")
-	driverCmd.PersistentFlags().StringVarP(&nodeID, "node-id", "n", nodeID, "identity of the node in which direct-csi is running")
-	driverCmd.PersistentFlags().StringVarP(&rack, "rack", "", rack, "identity of the rack in which this direct-csi is running")
-	driverCmd.PersistentFlags().StringVarP(&zone, "zone", "", zone, "identity of the zone in which this direct-csi is running")
-	driverCmd.PersistentFlags().StringVarP(&region, "region", "", region, "identity of the region in which this direct-csi is running")
-	driverCmd.PersistentFlags().BoolVarP(&controller, "controller", "", controller, "running in controller mode")
-	driverCmd.PersistentFlags().BoolVarP(&driver, "driver", "", driver, "run in driver mode")
+	driverCmd.Flags().StringVarP(&identity, "identity", "i", identity, "identity of this direct-csi")
```

Comment:
Persistent flags carry over to the subcommands as well. These flags are not persistentflags, so Ive made them normal flags